src/main/java/icons/MacheteIcons.java

```diff
@@ -0,0 +1,9 @@
+package icons;
```

Comment:
TBD with @mkondratek whether files like this one should be left in the root project or rather go into `frontendIcons` (as on #272)

Reply:
I would suggest to move it now. As a separate module it is more useful.

Reply:
see #272

Reply:
Ok, I moved it to `frontendIcons` module, but according to JB SDK (https://www.jetbrains.org/intellij/sdk/docs/reference_guide/work_with_icons_and_images.html) this class must be in `icons` package to can be simply used in `plugin.xml`